google-cloud-vm_migration-v1/lib/google/cloud/vm_migration/v1/vm_migration/paths.rb

```diff
@@ -0,0 +1,216 @@
+# frozen_string_literal: true
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Auto-generated by gapic-generator-ruby. DO NOT EDIT!
+
+
+module Google
+  module Cloud
+    module VMMigration
+      module V1
+        module VMMigration
+          # Path helper methods for the VmMigration API.
```

Comment:
I think we've discussed this before, but just noting that the API "name" here looks a bit strange since it does not match the module name (and it's not the actual product name).

Reply:
Hmm, yeah, we should get that fixed in the generator.